tests/test_full.py

```diff
@@ -41,10 +41,26 @@ def test_full():
                 2.3250037768187326,
                 48.86618482685007
             ],
-            "type": "Point"
+            "type": "Point",
+            "center": [
+                2.3250037768187326,
+                48.86618482685007
+            ]
         },
+        "label": "62B Rue de Lille (Paris)",
         "address": {
-            "label": "62B Rue de Lille (Paris)"
+            "admins": [],
+            "id": "addr:2.326285;48.859635",
+            "name": "62B Rue de Lille",
+            "house_number": "62B",
+            "label": "62B Rue de Lille (Paris)",
+            "postcode": "75007;75008",
```

Comment:
We would expect a single postcode in this case, with a specific address with house number.  
When reading this test, it's hard to say if this is related to the implementation or due to an unrealistic fixture.